benchmarks/python/test_softmax_fwd.py

```diff
@@ -82,6 +87,11 @@ def test_softmax_fwd_nvf_benchmark(
 @pytest.mark.parametrize("size", generate_input_sizes(dims=2))
 @pytest.mark.parametrize("dtype", FLOAT_DTYPES)
 @pytest.mark.parametrize("reduction_axis", [0, 1])
+@pytest.mark.inner_persistent
+@pytest.mark.outer_persistent
```

Comment:
similarly, inner or outer persistent depends on reduction axis.

Reply:
So does 1 determined inner reduction?

Reply:
yes

Reply:
This test takes a 2D input, dim-1 is the inner dim and dim-0 is the outer dim. If we do reduction on dim-1, we call it inner reduction.